Rocket.Chat/Views/Chat/RCTextView.swift

```diff
@@ -169,7 +169,7 @@ extension RCTextView: UITextViewDelegate {
                 return false
             }
 
-            ChatViewController.shared?.presentActionSheetForUser(user, source: (textView, textView.firstRect(for: range)))
+//            ChatViewController.shared?.presentActionSheetForUser(user, source: (textView, textView.firstRect(for: range)))
```

Comment:
Regression: not opening action sheet on tapping mention

Reply:
✅